openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -17,7 +18,6 @@
 
 
 @skip_unless_lms
-@mock.patch(UTILS_MODULE + '.get_api_data')
```

Comment:
The new unit tests I am adding don't require this patch. I added it to the specific tests that require it.